README.md

```diff
@@ -111,14 +111,13 @@ Use "New pull request button" at https://github.com/edu-xored/memorise
  
 ### Ask other members to review your changes  
 
-if all comments answered and all problems resolved some one who is resonsible (lead of the group) submits the chages to master repository
+if all comments answered and all problems resolved some one who is responsible (lead of the group) submits the changes to master repository
 ### Only after that you close the issue 
 Be close make you you have done:
 
 * Provide basic case for you issue to reproduce at web front (like product owner case) 
-* Provide unit tests for algoriths 
-* Provide integration test for new components like crouwler 
-* Provide description of some other cases and change you have made in therms of Quality Assurants Engineer (I've changed the logic of memos searching)
+* Provide unit tests for algorithms 
+* Provide integration test for new components like crowler 
```

Comment:
I think it should be "crawler" instead of "crowler". This applies to every "crowler" usage in README file.

Examples:
- [wiki: web crawler](https://en.wikipedia.org/wiki/Web_crawler)
- [crawler4j](https://github.com/yasserg/crawler4j)